src/doc/en/developer/walkthrough.rst

```diff
@@ -3,54 +3,40 @@
-Sage Development Process
+Development Walk-through
 ========================
 
-.. WARNING::
+This section is a concise overview of the Sage development process. We will see
+how to make changes to the Sage source code and record them in the Git revision
+control system.
 
-    **Sage development moved to GitHub in February 2023.** After the transition,
-    some parts of this guide (especially those related with `the Sage Trac
-    server <https://trac.sagemath.org>`_) became obsolete and need to be
-    updated according to the new workflow on GitHub. See our `transition guide
-    from Trac to GitHub
-    <https://github.com/sagemath/trac-to-github/blob/master/docs/Migration-Trac-to-Github.md>`_
-    for the preliminary version of the workflow.
-
-This section is a concise overview of the Sage development process. In
-it, we will see how to make changes to the Sage source code and record
-them in the ``git`` revision control system.
-
-We also have a handy `one-page "cheat sheet"
+We also have a handy `one-page cheat sheet
 <http://github.com/sagemath/git-trac-command/raw/master/doc/git-cheat-sheet.pdf>`_
 of commonly used git commands that you can print out and leave on your
 desk.  We have some :ref:`recommended references and tutorials
 <section-git-tutorials>` as well.
 
-In the following sections on :ref:`chapter-sage-trac` and
-:ref:`section-git-tricks-and-tips` we will look at communicating these
-changes back to the Sage project.  All changes to Sage source code
-have to go through the `Sage Trac development server
-<https://trac.sagemath.org>`_.
-
-As an alternative to using the Trac server directly, you can fork and
-create a Merge Request (MR) at `GitLab <https://gitlab.com/sagemath/sage>`_
-which will automatically fetch your code and open a ticket on our trac
-server.
+In the sections of the following chapter :ref:`section-development-on-github`,
+we will look at communicating these changes back to the Sage project. All
+changes to Sage source code have to go through `the Sage repository
+<https://github.com/sagemath/sage>`_ on GitHub.
 
-Pull Requests (PR) on GitHub are currently not supported by the
-SageMath project.
+Git is a tool to exchange commits (organized into branches) with other
+developers. As a distributed revision control system, it does not have the
+notion of a central server. `The Sage repository
+<https://github.com/sagemath/sage>`_ on GitHub is just one of many possible
+remote repositories from your (or rather your Git) point of view.
 
+For examples, we assume your name Alice. Always replace it with your own name.
 
 .. _section-walkthrough-setup-git:
 
-Configuring Git
-===============
+Checking Git
+============
 
-One way or another, ``git`` is what Sage uses for tracking changes.
-So first, open a shell (for instance, Terminal on Mac) and check that
-``git`` works::
+First, open a shell (for instance, Terminal on Mac) and check that Git works::
 
-    [user@localhost]$ git
+    [alice@localhost ~]$ git
```

Comment:
The file begins with `.. highlight:: shell-session`, which set `shell-session` as default language.

Reply:
Then something is not working with the copy hover. If I use this to copy the command, the whole display text is copied. @mkoeppe do you have an idea?

Reply:
In the last commit, I replaced `.. highlight:: shell-session` with `.. highlight:: console` in `walkthrough.rst`. For me, they seem the same. Let me know if this works for you.

Reply:
According to chatGPT:

In Sphinx RST, the directives .. highlight:: console and .. highlight:: shell-session are both used to highlight console or terminal commands in code blocks, but there is a subtle difference between them.

The .. highlight:: console directive is used to highlight generic console or command-line commands. This directive will not include the prompt symbol ($ or >) at the beginning of each command line. This can be useful for highlighting commands that are not specific to a particular shell or terminal emulator.

On the other hand, the .. highlight:: shell-session directive is used to highlight console or terminal commands along with the prompt symbol. This is useful for showing a complete session, including both the commands entered and the output returned by the terminal. The shell-session directive is typically used when demonstrating how to use a specific shell or terminal emulator.

In summary, .. highlight:: console is used to highlight generic console or command-line commands, while .. highlight:: shell-session is used to highlight a complete console or terminal session, including the prompt symbol.

Reply:
It's possible that copybutton needs to be configured to recognize these more elaborate prompts; see https://sphinx-copybutton.readthedocs.io/en/latest/use.html#using-regexp-prompt-identifiers

Currently it only recognizes `sage: `, `.... `, and plain `$`.

Reply:
If ".. highlight:: console" doesn't work for you either, then I will revert back to ".. highlight:: shell-session".